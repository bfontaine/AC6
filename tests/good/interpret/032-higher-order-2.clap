def compose (f) (g) (x) = f (g x)
val z = compose (fun (x) => x * 2) (fun (x) => x + 1) 20

