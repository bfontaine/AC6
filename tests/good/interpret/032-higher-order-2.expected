:- compose = <code>
:- z = 42
